### PR TITLE
widgets: Implement bindings for FilePicker

### PR DESCRIPTION
**Pull request by trufae**

### CODE REVIEW

**Comment by kenz-gelsoft** [comment]
Thank you for your contributing.

BTW, this generated wx::FilePickerCtrl class is somewhat tested on your project or any other test code?

If somewhat tested as usable in your (private) project, it is OK for now (because almost all other classes is not well tested as well).

File: wx-core/src/lib.rs [hunk 1]
```diff
@@ -718,10 +797,6 @@ impl<'a, P: WindowMethods> ComboBoxBuilder<'a, P> {
         self.id = id;
         self
     }
-    pub fn value(&mut self, value: &str) -> &mut Self {
-        self.value = value.to_string();
-        self
-    }
```
Comment (kenz-gelsoft) [inline wx-core/src/lib.rs:724]: Why adding support to wxFilePickerCtrl removes other class(here, ComboBoxCtrl)'s builder method?

Is this mistake? If so, please fix this.

File: codegen2.py [hunk 2]
```diff
@@ -146,6 +146,7 @@ def generated_h(classes, libname):
 #include <wx/clrpicker.h>
 #include <wx/datectrl.h>
 #include <wx/filepicker.h>
+#include <wx/filedlg.h>
```
Comment (kenz-gelsoft) [inline codegen2.py:149]: Please insert this alphabetical order if you don't strong preference to this order.

File: doxybindgen.toml [hunk 3]
```diff
@@ -18,6 +18,7 @@ wxml_files = [
     'wxml/classwx_date_picker_ctrl.xml',
     'wxml/classwx_date_time.xml',
     'wxml/classwx_dir_picker_ctrl.xml',
+    'wxml/classwx_file_picker_ctrl.xml',
```
Comment (kenz-gelsoft) [inline doxybindgen.toml:21]: Please insert this in alphabetical order too.

File: wx-core/include/generated.h [hunk 4]
```diff
@@ -6,6 +6,7 @@
 #include <wx/clrpicker.h>
 #include <wx/datectrl.h>
 #include <wx/filepicker.h>
+#include <wx/filedlg.h>
```
Comment (kenz-gelsoft) [inline wx-core/include/generated.h:9]: This too.

**Comment by kenz-gelsoft** [comment]
- Checked generated code are as-is generated.
- Once `lib.rs` diff is checked as sane, I'll merge this.

**Comment by kenz-gelsoft** [comment]
Thank you for considering contribution for my project.

I'm sorry, but after PR submitted, I've changed codegen, so fixing conflict is not easy.

I'll add wxFilePickerCtrl support in my next PR, closing this.

**Comment by trufae** [comment]
Unfortunely i didnt had the time to come back to this pr in time. Thanks for caring about it and keeping up with the project. Looking forward to use it in the future 👍

**Pull request closed without merging**